tools/changelog.py

```diff
@@ -111,7 +113,7 @@ def main(token, revision_range):
     # document authors
     authors = get_authors(revision_range)
     heading = u"Contributors"
-    print()
+    print(u"")
```

Comment:
Does this behave differently?

Reply:
I thought it did, but it looks like the problem was an artifact of the UTF8Writer.